Sources/Private/MainThread/NodeRenderSystem/Nodes/RenderNodes/GradientFillNode.swift

```diff
@@ -27,7 +27,7 @@ final class GradientFillProperties: NodePropertyMap, KeypathSearchable {
       PropertyName.opacity.rawValue : opacity,
       "Start Point" : startPoint,
       "End Point" : endPoint,
```

Comment:
Should we also add these 2 to the `PropertyName` enum for consistency?

Reply:
For now I've only been defining properties in the `PropertyName` enum once they're used by both rendering engines. There's a ton of string values currently defined throughout the main thread rendering engine and it seems like a good idea to avoid touching them until / unless necessary